batch/batch/cloud/gcp/resource_utils.py

```diff
@@ -17,6 +18,7 @@
 
 
 gcp_valid_machine_types = []
+gcp_valid_machine_types.append("g2-standard-4")
```

Comment:
@sjparsa, I'll approve anyway but can you follow up with a nit PR that just changes this to `gcp_valid_machine_types = ['g2-standard-4']`?